webpack.config.js

```diff
@@ -344,14 +344,14 @@ function getWebviewsConfig(mode, env) {
 		},
 		mode: mode,
 		target: 'web',
-		devtool: mode === 'production' ? false : 'source-map',
+		devtool: mode === 'production' ? false : 'inline-source-map',
```

Comment:
I haven't tried this (and won't be able to), but please make sure this doesn't negatively impact normal webview debugging.

Reply:
I don't see any impact on the webview debugging. As it works using 'source-map' as well, I can revert that line if you prefer.

Reply:
Changed in commit: `Changed "inline-source-map" to "source-map"`.